iaso/tests/api/test_check_version.py

```diff
@@ -16,79 +19,79 @@ def setUpTestData(cls):
         )
 
     def test_check_version_delete(self):
-        """DELETE /api/mobile/checkversion should result in a 405 response"""
+        """DELETE /api/mobile/checkversions should result in a 405 response"""
```

Comment:
Why is this still plural?

This applies to the whole file.

Reply:
corrected